sdk/samples/iot/README.md

```diff
@@ -341,7 +341,7 @@ Set the following environment variables for all samples:
 
 Set the following environment variables if running any of these samples: `paho_iot_hub_c2d_sample`, `paho_iot_hub_methods_sample`, `paho_iot_hub_telemetry_sample`, `paho_iot_hub_twin_sample`, `paho_iot_pnp_sample`, `paho_iot_pnp_component_sample`
 
-<details><summary><i>Instructions to set environment variables for IoT Hub X.509 Certificate samples:</i></summary>
+<summary><i>Instructions to set environment variables for IoT Hub X.509 Certificate samples:</i></summary>
```

Comment:
```suggestion
##### Set environment variables for IoT Hub X.509 Certificate samples
```

Reply:
and remove `<p>` tag below. All `<p>` tags can be removed in this document assuming proper indentation

Reply:
I see multiple places in this document and also before the instruction. do I need to remove whatever is there before instruction?

Reply:
Modified